test_runner/build.gradle.kts

```diff
@@ -258,3 +259,17 @@ tasks.assemble {
     dependsOn(generateManpageAsciiDoc)
 }
 // end --- ASCII doc generation ---
+
+val updateVersion by tasks.registering {
+    shouldRunAfter(tasks.processResources)
+    if (!runningOnBitrise) doLast {
+        File("${project.buildDir}/resources/main/version.txt").writeText("local_snapshot")
+        File("${project.buildDir}/resources/main/revision.txt").writeText(
+            String(Runtime.getRuntime().exec("git rev-parse HEAD").inputStream.readBytes())
```

Comment:
should we exec on a background thread? here's an example:
https://github.com/bootstraponline/gradle_enterprise_example/blob/master/app/enterprise.gradle.kts#L152

also probably better to use `exec { ... }` instead of Runtime directly.

Reply:
Running on background from your example requires GradleEnterprisePlugin, TBH I have no experience with build scan. Any way, adding GradleEnterprisePlugin looks like standalone tasks, not a part of another. Also Updating those two files is pretty fast so it don't much impact on build time. 

Ok, I will use `exec { ... }`.